hackathon/urls.py

```diff
@@ -9,5 +15,8 @@
     path("create_hackathon", create_hackathon, name='create_hackathon'),
     path("<int:hackathon_id>/update_hackathon", update_hackathon, name="update_hackathon"),
     path("<int:hackathon_id>/delete_hackathon", delete_hackathon, name="delete_hackathon"),
+    path('<int:pk>/',
+         HackathonDetailView.as_view(), name='hackathon-detail'),
+    path('ajax/enroll/', ajax_enroll_toggle, name='enroll-toggle')
```

Comment:
Not sure you need to prefix with `ajax` here. I would just keep enroll toggle. Also, please keep with the naming convention and use underscores instead of hypens.